.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
I think this file should be left to the user to add if they plan to use nix. I would not want to recommend this for all of our users that want to build in this repo.